packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -17,11 +17,13 @@ import type {
   BorderWidthValues,
   SpacingValues,
 } from '../../../tokens/theme/theme.d';
-import { ButtonSpinner } from '../ButtonSpinner';
```

Comment:
Yup ^

Reply:
but that was raised last time too to inline it and we had reasons to not inline it. what changed?

Reply:
Last time it was increasing the complexity for handling the children logic since we didn't have the Box component. Having a Box component makes it feasible to do this.